smarts/core/scenario.py

```diff
@@ -530,8 +527,8 @@ def discover_traffic_histories(scenario_root):
             return []
 
         traffic_histories = []
-        with open(path, "rb") as f:
-            files = pickle.load(f)
+        with open(path, "rb") as inf:
+            files = pickle.load(inf)
```

Comment:
just curious what is `inf` an abbreviation for? My first reaction is "infinity"....

Reply:
My guess is in_file, although `inf` does have that special meaning for numbers.

Reply:
Yeah, Tucker is right.  It was supposed to be "input_file".  The reason for the change was because `f` was being used in the list comprehension immediately following this code, so I wanted to avoid any possible confusion.